From bf7ec52eaafbb67385bf8ee0da07c4310c9bc8c2 Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@intevation.de>
Date: Sun, 28 Aug 2011 13:27:09 +0200
Subject: [PATCH] Libbfd needs zlib. Expand link libraries.

    This is a fix to a evil part of cmake code. Libraries
    are added without a check and build fails if they are not
    there. This needs to be fixed properly but only affects
    kde-windows mingw builds.
---
 drkonqi/kdbgwin/CMakeLists.txt |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/drkonqi/kdbgwin/CMakeLists.txt b/drkonqi/kdbgwin/CMakeLists.txt
index dcc6025..e3191b8 100644
--- a/drkonqi/kdbgwin/CMakeLists.txt
+++ b/drkonqi/kdbgwin/CMakeLists.txt
@@ -16,8 +16,10 @@ set(kdbgwin_SRCS
 set(COMMON_LIBS ${KDE4_KDECORE_LIBS} ${KDE4_KDEUI_LIBS} dbghelp psapi shlwapi)
 
 if ( MINGW )
+    # FIXME this should be handled in a higher level cmake with proper find
+    # scripts and configuration
     kde4_add_executable(kdbgwin ${kdbgwin_SRCS} mingw_generator.h mingw_generator.cpp)
-    target_link_libraries(kdbgwin ${COMMON_LIBS} bfd iberty intl)
+    target_link_libraries(kdbgwin ${COMMON_LIBS} bfd iberty intl zlib)
 endif ( MINGW )
 if ( MSVC )
     kde4_add_executable(kdbgwin ${kdbgwin_SRCS} msvc_generator.h msvc_generator.cpp)
-- 
1.7.2.5

